### ISSUE TITLE
Meeting agenda empty state

### ISSUE THREAD

**Issue by jordanh** [ParabolInc/action#846]
## Issue - Enhancement

Going through some user feedback notes, I see that there are a few points of feedback like, "I'm not sure what this Agenda list is even for," when folks look at the dashboard.

Let's add some empty state content to clarify what the meeting agenda is for.

- **Estimated effort:** 1 + 10 bonus points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by jordanh** [comment]
Here's a quick take:

```
Add a reminder to be discussed with others during the next meeting. Use a few short words to jog your memory like, "upcoming vacation."

Start or schedule a meeting __here__.
```

What do you think? Safe to try? Or, do you have a better idea?

**Comment by ackernaut** [comment]
Closed via #902
